finish/system/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java

```diff
@@ -56,59 +56,39 @@ public boolean verify(String hostname, SSLSession session) {
                     .build();
     }
 
-    @After
+    @AfterEach
     public void teardown() {
         client.close();
     }
     
     @Test
+    @Order(1)
     public void testPodNameNotNull() {
         response = this.getResponse(clusterUrl);
         this.assertResponse(clusterUrl, response);
         String greeting = response.getHeaderString("X-Pod-Name");
         
-        assertNotNull(
-            "Container name should not be null but it was. The service is probably not running inside a container",
-            greeting);
+        assertNotNull(greeting,
+            "Container name should not be null but it was. The service is probably not running inside a container");
+    }
+
+    private void assertResponse(String clusterUrl2, Response response2) {
```

Comment:
- [ ] `assertResponse` function is blank
- [ ] `getResponse` function is returning null


these both should be like [this](https://github.com/OpenLiberty/guide-istio-intro/blob/master/finish/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java#L116)

Reply:
addressed.